testclient/mainwindow.cpp

```diff
@@ -243,15 +251,10 @@ void MainWindow::printPBClicked()
 
     case DISCONNECTED:
         QMessageBox::information(this, tr("Error"), tr("Not Connected To a Printer"));
-        if(!core->pluginLoaded()){
-            addLog(tr("No plugin loaded !"));
-            addLog(tr("Requesting Firmware..."));
-            core->requestFirmware();
-        }
         break;
```

Comment:
Why remove this ?
It's good to try requestFirmware when print is clicked and no FW is detected